website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -39,6 +39,16 @@ You can read more about each of these behavior changes in the following links:
 
 - (Introduced, disabled by default) [`validate_macro_args`](/reference/global-configs/behavior-changes#macro-argument-validation). If the flag is set to `True`, dbt will raise a warning if the argument `type` names you've added in your macro YAMLs don't match the argument names in your macro or if the argument types aren't valid according to the [supported types](/reference/resource-properties/arguments#supported-types).
 
+### Integrating dbt Core artifacts with dbt Cloud projects
```

Comment:
can we move this up - so that the "Managing changes to legacy behaviors" heading is the last one in this section?

Reply:
no worries!

Reply:
moved it up!